### ISSUE TITLE
Synchronize path between panels

### ISSUE THREAD

**Issue by jlx84** [MeanEYE/Sunflower#150]
Could you please add a button or static bookmark entry which could synchronize path between left panel and right panel. Nice file manager btw., I used Altap Salamander on Windows and Sunflower suits me best on Linux.

**Comment by ArseniyK** [comment]
You mean open same dir on oposite panel? There shortcut fot it Ctrl+left/right.

**Comment by MeanEYE** [comment]
We do have shortcut but not a menu item for it. We should put that in plugin menu.

**Comment by jlx84** [comment]
Yes, that does the trick, thanks! But still, a menu item would be nice for new users who are not familiar with shortcuts yet.
